lib/models/download.js

```diff
@@ -20,7 +46,8 @@ const schema = new mongoose.Schema({
   url: { type: String },
   upload: uploadSchema,
   pipelines: { type: String },
-  isPublic: { type: Boolean, default: false }
+  isPublic: { type: Boolean, default: false },
+  expireTTL: { type: Date, default: undefined }
```

Comment:
This should probably be called `expirationDate` since `TTL` is usually a number not a date.

Reply:
https://github.com/LearningLocker/learninglocker/commit/70130390a6b79714a30e939a37b685fdea7131df fixed